packages/openneuro-server/src/graphql/resolvers/comment.js

```diff
@@ -18,6 +18,34 @@ const replies = obj => {
   return Comment.find({ parentId: obj._id }).exec()
 }
 
+/**
+ * Flattens an array of arrays
+ * @param {*[][]} arr
+ * @returns {*[]}
+ */
+export const flatten = arr => [].concat(...arr)
+
+/**
+ * @typedef {Object} Comment
+ * @property {string} _id
+ */
+
+/**
+ * returns a flat array of all the dependencies of the given comment
+ * @param {Comment} obj
+ * @returns {Promise<Comment[]>}
+ */
+const allNestedReplies = async obj => {
+  const replies = await Comment.find({ parentId: obj._id }).exec()
+  if (!replies.length) {
+    return replies
+  } else {
+    let nestedReplies = await Promise.all(replies.map(allNestedReplies))
+    const r = flatten([replies, ...nestedReplies])
```

Comment:
nestedReplies can be const here and I'd just return the next line instead of creating a single character variable.